source/game/kart/KartMove.hh

```diff
@@ -5,11 +5,14 @@
 
 namespace Kart {
 
+class KartJump;
```

Comment:
Does this have to be forward declared due to cyclical dependencies? If so, I would forward declare KartMove in KartJump.hh.